opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledge.jsx

```diff
@@ -46,46 +39,52 @@ export const groupingKnowledgeAttackPatternsGraphQuery = graphql`
         x_opencti_order
         x_opencti_color
       }
-      objects(all: true, types: ["Attack-Pattern"]) {
-        edges {
-          node {
-            ... on AttackPattern {
-              id
-              entity_type
-              parent_types
-              name
-              description
-              x_mitre_platforms
-              x_mitre_permissions_required
-              x_mitre_id
-              x_mitre_detection
-              isSubAttackPattern
-              parentAttackPatterns {
-                edges {
-                  node {
-                    id
-                    name
-                    description
-                    x_mitre_id
-                  }
+      ...GroupingKnowledge_fragment
+    }
+  }
+`;
+
+const GroupingAttackPatternsFragment = graphql`
+  fragment GroupingKnowledge_fragment on Grouping {
```

Comment:
maybe rename this fragment in a more precise way... (like GroupingKnowledgeAttackPattern_fragment)

Reply:
done